data_structures/queue_with_stacks/queue_with_stacks.py

```
@@ -0,0 +1,34 @@
+from .stack import Stack
+
+
+class Queue:
+    """class for Queue"""
+    def __init__(self, iter=[]):
+        self.stack1 = Stack()
+        self.stack2 = Stack()
+        self._len = 0
+
+        for item in iter:
+            self.enqueue(item)
+
+    def enqueue(self, val):
+        """add new iten to the queue"""
+        if val:
+            self.stack1.push(val)
+            self._len += 1
+            return self.stack1
+
+        return False
+
+    def dequeue(self):
+        """remove item from the front"""
+        if self._len == 0:
+            return False
```

Instead of returning `False`, it would be better to raise an exception.